### ISSUE TITLE
externalTargets not working as expected with hostnames resolving to multiple ip addresses

### ISSUE THREAD

**Issue by vburckhardt** [pingcap/chaos-mesh#692]
## Bug Report

**What version of Kubernetes are you using?**
1.16

**What did you do?**

network chaos:
```
spec:
  action: delay
  delay:
    correlation: "0"
    jitter: 0ms
    latency: 10000ms
  direction: to
  duration: 59s
  externalTargets:
  - www.google.com
  mode: all
  scheduler:
    cron: '@every 1m'
  selector:
    labelSelectors:
      app: microserviceb
    namespaces:
    - default
  value: ""
```

**What did you expect to see?**

kube exec inside the pod, and run ping www.google.com
expecting to see ping delayed consistently for 59s, not delayed for 1s, and so on

**What did you see instead?**

ping delayed only from time to time, inconsistently.

**Comment by vburckhardt** [comment]
I believe the root of the issue is on that line, returning only one ip from the ns lookup, instead of all of them.  https://github.com/pingcap/chaos-mesh/blob/716d54ed33bdb979db485e2f57f877e3a2772764/controllers/networkchaos/netem/utils.go#L38

A workaround would be to specify multiple ips instead of an host (using ips returned by nslookup)

Also seeing from the logs (notice, one single ip per minute)
    Jul 13 22:12:24 chaos-daemon-nxwg2 chaos-daemon INFO chaos-daemon-server	add CIDR to ipset	{"command": "/usr/bin/nsenter -n/proc/9281/ns/net -- ipset add block_591b82551a8bd1a_netemold 216.58.217.36/32"}
    Jul 13 22:13:24 chaos-daemon-nxwg2 chaos-daemon INFO chaos-daemon-server	add CIDR to ipset	{"command": "/usr/bin/nsenter -n/proc/9281/ns/net -- ipset add block_591b82551a8bd1a_netemold 172.217.14.196/32"}
    Jul 13 22:14:24 chaos-daemon-nxwg2 chaos-daemon INFO chaos-daemon-server	add CIDR to ipset	{"command": "/usr/bin/nsenter -n/proc/9281/ns/net -- ipset add block_591b82551a8bd1a_netemold 172.217.14.196/32"}
    Jul 13 22:15:24 chaos-daemon-nxwg2 chaos-daemon INFO chaos-daemon-server	add CIDR to ipset	{"command": "/usr/bin/nsenter -n/proc/9281/ns/net -- ipset add block_591b82551a8bd1a_netemold 172.217.14.196/32"}
    Jul 13 22:16:24 chaos-daemon-nxwg2 chaos-daemon INFO chaos-daemon-server	add CIDR to ipset	{"command": "/usr/bin/nsenter -n/proc/9281/ns/net -- ipset add block_591b82551a8bd1a_netemold 216.58.217.36/32"}

nslookup www.google.com returns 6 ipv4 on my side, so 1 out of 6 chances to hit the delay depending on which ip ping selects.

**Comment by YangKeao** [comment]
Oops, sorry for the silly mistake :cry: . I have fixed it in #693
